src/components/organisms/AdminProfileModal/AdminProfileModal.tsx

```diff
@@ -0,0 +1,136 @@
+              <div className="sm:flex sm:items-start">
+                <div className="mt-3 text-left sm:mt-0">
+                  <Dialog.Title
+                    as="h3"
+                    className="text-lg leading-6 font-medium text-gray-900"
+                  >
+                    My profile
+                  </Dialog.Title>
+                  <div>
+                    <div className="mt-7 mb-5 flex flex-row gap-3">
+                      <div className="flex-shrink-0 h-20 w-20">
+                        <img
+                          alt=""
+                          className="rounded-full w-20 h-20"
+                          src="https://images.unsplash.com/photo-1504257432389-52343af06ae3?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=987&q=80"
+                        />
+                      </div>
+                      <div className="flex flex-col">
+                        <h2 className="font-sm">Tom Cook</h2>
+                        <p className="font-sm text-gray-700">
+                          tomcook@hotmail.co.uk
+                        </p>
+                        <span className="font-xs">
+                          is busy{" "}
+                          <a href="/" className="underline font-xs">
+                            Change status
+                          </a>
+                        </span>
+                      </div>
+                    </div>
+                    <div>
+                      <span>What level of English do you have?</span>
+                      <p>Excellent!</p>
+                    </div>
+                    <div>
+                      <span>Where are you from?</span>
+                      <p>San Francisco</p>
+                    </div>
+                  </div>
+                </div>
+              </div>
+              <div className="mt-5 sm:mt-4 sm:flex sm:flex-row-reverse">
+                <button
+                  type="button"
+                  className="w-full inline-flex justify-center rounded-md border border-transparent shadow-sm px-4 py-2 bg-sparkle text-base font-medium text-white hover:bg-sparkle-darker focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-red-500 sm:ml-3 sm:w-auto sm:text-sm"
+                  onClick={hideHandler}
+                >
+                  Edit profile
+                </button>
+                <button
+                  type="button"
+                  className="mt-3 w-full inline-flex justify-center rounded-md border border-gray-300 shadow-sm px-4 py-2 bg-white text-base font-medium text-gray-700 hover:bg-gray-50 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500 sm:mt-0 sm:w-auto sm:text-sm"
+                  onClick={hideHandler}
+                  ref={cancelButtonRef}
+                >
+                  Cancel
+                </button>
```

Comment:
Extract this out into a button component :)

Reply:
created a raw AdminButton component: https://github.com/sparkletown/sparkle/pull/2812/commits/9f0e19504410650d3f6936c81dfbd764241fc028